app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -247,6 +251,8 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
         }
       }
     }
+    if (vm.catalogItemModel.provisioning_playbook_ttl !== undefined)
```

Comment:
Can the value be `undefined` for `vm.catalogItemModel.provisioning_playbook_ttl`?
I guess that depends on the .rb code below? -
```
playbook_details[:provisioning][:playbook_ttl] = provision[:playbook_ttl]
```

If you don't expect the value to be `undefined`, then you can skip this conditional.

(Also preferred syntax for `if` per the styling guide is -
```javascript
if (condition) {
  // code below
  ...
  ...
}
```